src/writeckpt.cpp

```diff
@@ -137,13 +137,51 @@ mtcp_writememoryareas(int fd)
     }
   }
 
+  // FIXME:  We need a comment here.  procSelfMaps was created above,
+  //         with local scope so that it will be destroyed on return.
+  //         But we test it here, because maybe it wasn't really created. ???
+  //         How could this happen?
```

Comment:
@karya0 , Thanks.  I'm renaming the local variable as `procSelfMapsTmp;`, so that there will be no confusion anymore.  With that clarification, I'm also deleting the FIXME comment.